UnleashClient/loader.py

```diff
@@ -11,7 +12,10 @@ def _create_strategies(provisioning: dict,
 
     for strategy in provisioning["strategies"]:
         try:
-            if "parameters" in strategy.keys():
+            if strategy['name'] == 'flexibleRollout':
```

Comment:
I might have been a bit unclear about this; Strategy constraints can be applied to all strategies, not just `FlexibleRollout`. For now it has only been made available via the `FlexibleRollout` strategy in the UI. This will change when the feature stabilises.

Reply:
Sorry, I didn't see this. :)

Understood!  I'm going to finish up the flexibleRollout work on this PR (which involves laying the groundwork for adding constraints to other strategies).  I'll work on another PR to update the other strategies.

Reply:
PR now supports constraints for all strategies! :)